src/Samples/EventBindingsAndBehaviors.Views/EventBindingsAndBehaviors.Views.csproj

```diff
@@ -2,11 +2,16 @@
 
   <PropertyGroup>
     <TargetFramework>netcoreapp3.0</TargetFramework>
+    <OutputType>Exe</OutputType>
     <UseWpf>true</UseWpf>
   </PropertyGroup>
 
   <ItemGroup>
     <PackageReference Include="Microsoft.Xaml.Behaviors.Wpf" Version="1.1.3" />
   </ItemGroup>
 
+  <ItemGroup>
+    <ProjectReference Include="..\EventBindingsAndBehaviors\EventBindingsAndBehaviors.fsproj" />
+  </ItemGroup>
+
 </Project>
```

Comment:
Not a comment for this file per se, but is there any reason there's no design-time VM for this sample?

Reply:
Oh, I mistakenly thought it didn't display any data from the model.  I just added it now.